modules/motions/utils/validateAddress.ts

```diff
@@ -0,0 +1,11 @@
+import { constants, utils } from 'ethers'
+
+export function validateAddress(value: string) {
+  if (!utils.isAddress(value)) {
+    return 'Address is not valid'
+  }
+
+  if (value.toLowerCase() === constants.AddressZero) {
+    return 'Address must not be zero address'
+  }
+}
```

Comment:
What do you think about [consistent-return](https://eslint.org/docs/latest/rules/consistent-return)?

Reply:
I don't like that rule, but we can enable it if you want. This will require a lot of refactoring afterwards though. For now I can add explicit return statement to validator functions now and create a task in Linear to add this rule, wdyt?

Reply:
I would suggest adding default return to all new validate*. This can be the empty string or null, whichever you think is more appropriate.